src/display/text_layer.js

```diff
@@ -75,7 +81,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
     textDiv.style.fontSize = fontHeight + 'px';
     textDiv.style.fontFamily = style.fontFamily;
 
-    textDiv.textContent = geom.str;
+    textDiv.textContent = replaceWhitespaceWithStandardSpace(geom.str);
```

Comment:
Could you add a comment about why we are doing that? There is actually a second reason `measureText` will replace all white spaces to SPACE anyway.